Meta/CMake/common_options.cmake

```diff
@@ -24,3 +24,5 @@ serenity_option(HACKSTUDIO_BUILD OFF CACHE BOOL "Automatically enabled when buil
 
 serenity_option(ENABLE_JAKT OFF CACHE BOOL "Enable building jakt files")
 serenity_option(JAKT_SOURCE_DIR "" CACHE STRING "Pre-existing jakt language source directory")
+
+serenity_option(SERENITY_CACHE_DIR "${PROJECT_BINARY_DIR}/../caches" CACHE STRING "Location of shared cache of downloaded files")
```

Comment:
With this option here, is this line in the super build still needed?
https://github.com/SerenityOS/serenity/blob/2334b4cebdcce1104d85afa75315c2721cc2df83/Meta/CMake/Superbuild/CMakeLists.txt#L48

Reply:
Good point, removed.